packages/stencil-component/test-data/ej-fra/ej-fra-malformed.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0' encoding='utf-8'?>
+<TEI>
```

Comment:
will need to be redone for dev.readalong-component (remove malformation, pass through `readalong align`, reintroduce malformation)

Reply:
Right. Sigh...